apps/web/pages/auth/setup.tsx

```diff
@@ -160,6 +168,7 @@ const SetupFormStep1 = (props: { setIsLoading: (val: boolean) => void }) => {
                   formMethods.setValue("password", e.target.value);
                   await formMethods.trigger("password");
                 }}
+                hintErrors={["caplow", "min", "num"]}
```

Comment:
New `hintErrors` property to check conditions based on the keys from custom issues registered by Zod `superRefine`